### ISSUE TITLE
[v3.0.0-0 version] stage mouse down event bug report.

### ISSUE THREAD

**Issue by mollerzhu** [konvajs/vue-konva#182]
When I create a stage and add mousedown event and console.log(e.target), it will give 2 result. The code is:

```
<template>
    <div>
        <v-stage :config="stageCon" @mousedown="stageMouseDown">
            <v-layer>
                <v-circle :config="circleCon"></v-circle>
            </v-layer>
        </v-stage>
    </div>
</template>
<script setup lang='ts'>
const stageCon = {
    width: 300,
    height: 200
}
const circleCon = {
    x: 100,
    y: 100,
    radius: 70,
    fill: "red",
    stroke: "black",
    strokeWidth: 4
}
function stageMouseDown(e: any) {
    console.log(e.target)
}
</script>
```

The console will outputs like that:
```
Stage {_id: 65, eventListeners: Object, attrs: Object, index: 0, _allEventListeners: null…}
-----------------------
<canvas width="2228" height="1536" style="padding: 0px; margin: 0px; border: 0px; background: transparent; position: absolute; top: 0px; left: 0px; width: 1114px; height: 768px; display: block;"></canvas>
```
An extra dom info! I think it is a bug.

**Comment by mollerzhu** [comment]
Vue2 version is OK, @3.0.0-0 has this bug.

**Comment by lavrton** [comment]
Thanks to @k1nz that bug should be fixed in `vue-konva@3.0.1`
